securedrop/tests/utils/env.py

```diff
@@ -59,6 +60,9 @@ def setup():
 
 
 def teardown():
+    for t in threading.enumerate():
```

Comment:
I know that you have a nice descriptive commit message :smile: and that we can git blame, but let's make sure this is as clear as possible for future developers (and future us) by adding a comment above this line explaining why we added this to the `teardown` (basically just what you have in the commit message).